enhancements/oc/must-gather.md

```diff
@@ -29,24 +29,24 @@ superseded-by:
 ## Summary
 
 To debug something broken in the cluster, it is important to have a single command for an unskilled customer to run that
-gathers all the information we may need to solve the problem.  If you're familiar with `sosreport`, the idea is to have
-that, but focused on a kuberentes cluster instead of a host.  We need avoid the versioning skew complexity and the scaling
-problems inherent in a shared repo with input from multiple products.
+gathers all the information we may need to solve the problem. If you're familiar with `sosreport`, the idea is to have
+that, but focused on a kuberentes cluster instead of a host. We need to avoid the versioning skew complexity and the
+scaling problems inherent in a shared repo with input from multiple products.
 
 ## Motivation
 
-Gather everything.  Software breaks and you aren't smart enough to know exactly what you need to debug the problem ahead of time.  You figure
-that out *after* you've debugged the problem.  This tool is about the first shotgun gathering so you only have to ask the
+Gather everything. Software breaks and you aren't smart enough to know exactly what you need to debug the problem ahead of time. You figure
+that out *after* you've debugged the problem. This tool is about the first shotgun gathering so you only have to ask the
 customer once.
```

Comment:
This isn't how troubleshooting works; You don't gather everything you need all at once! Doing this is often improbable, time consuming and very data intensive (you gather way too much data to process effectively). Positioning this tools as a 'silver bullet' set's the wrong expectation with customers and associates, which simply leads to a frustrating support experience.

The [wolf fencing algorithm](https://dl.acm.org/doi/pdf/10.1145/358690.358695) is as I understand it the standard for software debugging practices/procedures. In short debugging is a set of steps where in you continually bisect the problem with new/more data [collections] (that continually narrow in on where the issue is). 

must-gather is the first salvo into this endeavor, giving us enough data to start the bisecting process, and gather more information with 'more specialized scripts' (acting as a packaging/orchestration tool) or with other tools (like `sosreport`). 

I wonder if this section should be framed this way?

Reply:
@sferich888 I honestly like the idea you have presented here. However, there are times when either a customer wants to collect all possible debug data and attach it to the support ticket right at the time of opening one, or a support engineer asks for logs (often from as wide surface area as possible) during the initial comment(s). At times, it's the latter that causes the former.

I understand that the above paragraph is endlessly debatable.

> I wonder if this section should be framed this way?

I was only fixing typos in the first commit. Do you want me to do it in this PR or a separate one? I'd prefer separate.

Reply:
I agree with Eric here, I think the best solution here is to just drop the first sentence "Gather everything" since the rest explicitly sets must-gather as the first step to figure out what's going on in the cluster based on broad and high-level gatherers. Even though you can repeatedly use must-gather for more thorough debugging, you have to be explicit in the exact invocation parameters to guide it towards what you're looking for.